prerequisites/software.rst

```diff
@@ -163,30 +163,14 @@ installed with the Zammad-Package.
 --------------------
 
 Zammad will store all content in a Database.
-You can choose between the following database servers:
 
 * PostgreSQL 9.3+
-* MySQL 5.7+ / MariaDB 10.3+ (⚠️ deprecated with Zammad 7.0+)
 
 .. danger::
 
-   .. include:: /appendix/includes/mysql-deprication-note.rst
+   .. include:: /appendix/includes/mysql-deprecation-note.rst
 
-   .. include:: /appendix/includes/mysql-deprication-link.rst
-
-.. warning:: **Required configuration for MySQL/MariaDB:**
-
-   * Use ``UTF-8`` encoding - ``utf8mb4`` for example will fail!
-   * Set ``max_allowed_packet`` to a value larger than the default of 4 MB
-     (64 MB+ recommended).
-
-   You may also want to consider the following settings for your MySQL server::
-
-      innodb_file_format = Barracuda
-      innodb_file_per_table = on
-      innodb_default_row_format = dynamic
-      innodb_large_prefix = 1
-      innodb_file_format_max = Barracuda
```

Comment:
Do we really want to remove a software dependency during deprecation phase?
Zammad does still allow you to run the installer that way even through we don't hint users on how.

This is kinda a bad idea and not really consistent. It also will nuke those who have to migrate systems that are not based on psql and plan on switching later. I'm fine with reducing explicit hints etc but that one's essential to me.

Or I don't get the ideas and points.

Reply:
After rethinking, I agree with you on this one.

Reply:
PM decided that we are not going to support new installations on MySQL. The intention of this PR is / was to make that explicit. What's the point in listing MySQL as an option for new installations here and supporting people on that? That will only lead to more migration efforts later.

Reply:
Right. All hints and notes on how to install this have been removed.
But above does also affect updating instances. At least from my point of view.

I'm not happy with this face right now. Like absolutely not happy.
We don't even scream at those who ignore our warnings and discouragement. But now we wanna reduce documentation.

I'd say this particular documentation part should stay as long as we're still in the grace period.
If we can't agree on that, Gerit will have to take it to PM to get a decision.